ros-tutorials/lesson-0-introduction.md

```diff
@@ -1,24 +1,75 @@
+This introduction will set up your system for the rest of the tutorials. It will expose you to some of the software we use: Unix command line, Docker, and Git.
 
-Before continuing, if you have not used the command line before, complete this short tutorial:
+## Command Line
 
-[Command Line Basics](https://www.vikingcodeschool.com/web-development-basics/a-command-line-crash-course)
+Before continuing, if you have never used the Unix command line (or Terminal in Mac or Command Prompt in Windows), complete [this short tutorial](https://www.vikingcodeschool.com/web-development-basics/a-command-line-crash-course).
 
-To run ROS, we will be using Docker. This is a software that allows you to run programs inside an isolated environment. This means that you do not need to download anything but the docker client, and the particular image we will be using. First, download the appropriate docker client:
+## Docker Setup
+
+To run ROS, we will be using Docker. This is a software that allows you to run programs inside an isolated environment called a container. So, you only need to download the Docker client and then run our image. Our image includes all of the little pieces of software we need, rather than installing them individually. If you're curious, learn more [here](https://www.docker.com/resources/what-container) or [here](https://docs.docker.com/get-started/).
+
+Download and install the appropriate Docker client:
 
 * [Mac](https://docs.docker.com/docker-for-mac/install/)
-* [Windows (Home)](https://docs.docker.com/toolbox/toolbox_install_windows/)
-* [Windows (Other)](https://docs.docker.com/docker-for-windows/install/)
-* Which Windows do I have? Start > type "PC Info" > note if you have Windows 10 Home, Pro, Education, etc.
+* [Windows (Home)](https://docs.docker.com/docker-for-windows/install-windows-home/)
+* [Windows (Pro, Education, etc.)](https://docs.docker.com/docker-for-windows/install/)
+
+:information_source: Which Windows do I have? *Settings > System > About > look at "Edition"*<br>
+:warning: If you have Windows Home, make sure you follow the installation instructions exactly. Instead, we recommend upgrading to Windows Education using [Duke's free student license](https://software.duke.edu), though it is not required.
+
+Once the Docker client is installed, run it like you would any other app on your computer.
+
+## Starter Code and Git
+
+To share and manage versions of our source code, we use [Git](https://git-scm.com/). When you have a folder of code files for your project, Git calls it a repository. We're going to use Git now to get some starter code for these tutorials.
+
+Open up Terminal in Mac or Command Prompt in Windows. Type `git --version` and press Enter. If the command is not found, close the window and install Git from [here](https://git-scm.com/downloads). Then re-open Terminal or Command Prompt.
```

Comment:
This could just be my computer, since I've messed around with the system path, but Git doesn't work in cmd prompt, it only works in git bash. If it works on you computer then this should be fine, but otherwise git bash might be recommended. Also, not sure if you want to tell them to move to the directory they want to clone in, instead of just cloning in their home directory (I know if you install Pspice, your home directory gets changed to the Pspice install location). You could also tell them to open the directory in terminal using right click or something like that.

Reply:
Git works in command prompt if you've added it to your path, which is an option during installation (and seems to be the default, but I can make an explicit note if you'd like, just let me know.)

About the cloning directory, I figured I'd leave it up to them. This is just the documentation repo anyway.